Chore: Improve Spelling, Grammar, and Broken Links (#4506)
PostHog/posthog.com: contents/tutorials/how-to-capture-events-the-easy-way.md

![Using the toolbar](../images/tutorials/custom-events-the-easy-way/create-events-easy-way-posthog.png)

> When the toolbar is launched, you’ll see a floating PostHog logo in your browser. You can drag this around to unobscure elements of the site if needed, and when you click it, it will reveal a pop-out menu, as you can see above.

Author: ```suggestion
When the toolbar is launched, you’ll see a floating PostHog logo in your browser. You can drag the logo to reveal obscured elements of the site as needed, and on click it will reveal a pop-out menu, as pictured above.
```
